plugins/contracts/package.json

```diff
@@ -6,6 +6,7 @@
     "@leancodepl/contractsgenerator-typescript-schema": "1.1.0-alpha.0",
     "@leancodepl/contractsgenerator-typescript-types": "1.1.0-alpha.0",
     "@leancodepl/contractsgenerator-typescript-values": "1.1.0-alpha.0",
+    "lodash": "4.17.21",
```

Comment:
I think this should be more relaxed. Pretty sure we don't need this specific version of lodash.